guides/doc-Updating_Project/master.adoc

```diff
@@ -0,0 +1,18 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:updating:
+:context: updating
+
+ifdef::satellite[]
```

Comment:
I thought we've discussed that this whole guide was applicable to upstream except for the disconnected server.

Reply:
Max did highlight the above but I haven't received confirmation yet. We can finalize it here.

Reply:
Ack! :+1: